smartsim/_core/control/controller.py

```diff
@@ -209,34 +269,40 @@ def get_jobs(self) -> t.Dict[str, Job]:
         with JM_LOCK:
             return self._jobs.completed
 
-    def get_entity_status(self, entity: t.Union[SmartSimEntity, EntityList]) -> str:
+    def get_entity_status(
+        self, entity: t.Union[SmartSimEntity, EntitySequence[SmartSimEntity]]
```

Comment:
or, since we dropped python 3.7, can we swap to A | B syntax?

Reply:
> Should we add a _SmartSimEntity = t.Union[SmartSimEntity, EntitySequence[SmartSimEntity]] alias for brevity due to how often this union is repeated?

Absolutely, but I would argue that it is outside the scope of this ticket. I will put one up to do this

> or, since we dropped python 3.7, can we swap to A | B syntax?

Unfortunately, we cannot "out of the box" as the `A | B` syntax was introduced natively in python 3.10. That said, if we are willing to adopt the use of `from __future__ import annotations` and perhaps a adjust couple of mypy settings, it should be backward compatible! What would you think about adding a ticket for this work as well?